packages/node_modules/@webex/react-component-space-item/src/index.js

```diff
@@ -88,78 +65,150 @@ function SpaceItem({
     return false;
   }
 
+  function getTeamColor(hex, isDarkShade) {
```

Comment:
I'm guessing we're probably going to need this elsewhere. Should we add it to the helper functions?

Reply:
couldn't hurt, is there a specific file you're thinking of?